<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TracePolyLineDlg</class>
 <widget class="QDialog" name="TracePolyLineDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>106</width>
    <height>27</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Segmentation</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QCheckBox" name="cpuButton">
     <property name="text">
      <string>CPU</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="validButton">
     <property name="toolTip">
      <string>Confirm segmentation</string>
     </property>
     <property name="statusTip">
      <string>Confirm segmentation</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="icon">
      <iconset resource="../icones.qrc">
       <normaloff>:/CC/images/smallValidate.png</normaloff>:/CC/images/smallValidate.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="cancelButton">
     <property name="toolTip">
      <string>Cancel</string>
     </property>
     <property name="statusTip">
      <string>Cancel segentation</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset resource="../icones.qrc">
       <normaloff>:/CC/images/smallCancel.png</normaloff>:/CC/images/smallCancel.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionSetRectangularSelection">
   <property name="icon">
    <iconset resource="../icones.qrc">
     <normaloff>:/CC/images/smallRectangleSelect.png</normaloff>:/CC/images/smallRectangleSelect.png</iconset>
   </property>
   <property name="text">
    <string>Rectangular selection</string>
   </property>
   <property name="toolTip">
    <string>Activates rectangular selection</string>
   </property>
  </action>
  <action name="actionSetPolylineSelection">
   <property name="icon">
    <iconset resource="../icones.qrc">
     <normaloff>:/CC/images/smallPolygonSelect.png</normaloff>:/CC/images/smallPolygonSelect.png</iconset>
   </property>
   <property name="text">
    <string>Polygonal selection</string>
   </property>
   <property name="toolTip">
    <string>Activaites polyline selection</string>
   </property>
  </action>
  <action name="actionUseExistingPolyline">
   <property name="icon">
    <iconset resource="../icones.qrc">
     <normaloff>:/CC/images/dbPolylineSymbol.png</normaloff>:/CC/images/dbPolylineSymbol.png</iconset>
   </property>
   <property name="text">
    <string>Use existing polyline</string>
   </property>
   <property name="toolTip">
    <string>Import polyline from DB for segmentation</string>
   </property>
  </action>
  <action name="actionExportSegmentationPolyline">
   <property name="icon">
    <iconset resource="../icones.qrc">
     <normaloff>:/CC/images/ccSave.png</normaloff>:/CC/images/ccSave.png</iconset>
   </property>
   <property name="text">
    <string>Export segmentation polyline</string>
   </property>
   <property name="toolTip">
    <string>Export segmentation polyline as new entity</string>
   </property>
  </action>
  <action name="actionProjectPolyline">
   <property name="text">
    <string>Project Polyline Vertexes on Selected</string>
   </property>
   <property name="toolTip">
    <string>project polyline on selected items</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icones.qrc"/>
 </resources>
 <connections/>
</ui>
